### PR TITLE
51 refactor aggregates part of the documentation

### PR DESCRIPTION
**Pull request by Antonyjin**
I made some changes to the Aggregates field in the python's documentation

Splitting the main points and adding some examples to make it clear

### CODE REVIEW

**Comment by hoh** [comment]
Is the introduction not redundant with the "Object types" section ?
image

File: docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            {
+                'address': '0x...',
+                'types': ['AGGREGATE'],
+                'aggregate_keys': ['testkey'],
+            }
+        ]},
+        channel: 'security',
+    })
+})();
+```
```
Comment (hoh) [inline docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md:80]: This is Typescript inside the Python SDK section... :thinking:

File: docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+Inside most Messages, there is an **"address"** field. This is the address for which the message applies (for which address to apply this aggregate, who posted that item…).</br>
```
Comment (hoh) [inline docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md:1]: Should this be in the "Message types" section ?

File: docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md (hunk 1 above)
Comment (Antonyjin) [reply docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md:80]: Sorry for that i wrote the wrong code here

File: docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md (hunk 2 above)
Comment (Antonyjin) [reply docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md:1]: I wanted to give a little introduction before talking about the security key. That's why i put this

**Review by MHHukiewitz — APPROVED** [review]
The refactoring makes sense. Splitting security key and delegating access maybe not so much, but it is important to consider that the security key applies to other message types too, so it's fine :+1:

**Pull request merged**